tests/unit/handlers/mentionEachUser.test.ts

```diff
@@ -42,7 +43,7 @@ describe("Test mention each function", () => {
       (res) => res.json()
     );
     expect(textMessage.data.content).toBe(
-      "Sorry no user found under this role."
+      `Sorry no user found with <@&${roleId}> role.`
```

Comment:
This message is only shown when dev_titl is passed to true then how this test work?